src/main/java/com/jfrog/ide/common/configuration/JfrogCliServerConfig.java

```diff
@@ -36,15 +39,27 @@ public String getArtifactoryUrl() {
 
     @Override
     public String getUsername() {
+        if (isNotBlank(getAccessToken())) {
+            return "";
+        }
         return getValueFromJson(USER_NAME);
     }
 
     @Override
     public String getPassword() {
+        if (isNotBlank(getAccessToken())) {
+            return "";
+        }
         return getValueFromJson(PASSWORD);
     }
 
+    @Override
     public String getAccessToken() {
+        // If the configured access is refreshable, we should use
+        // the saved username and password in this context.
+        if (isNotBlank(getValueFromJson(REFRESH_TOKEN))) {
```

Comment:
I think that introducing the refreshable token logic to the IDEs is not needed. We can just pick up the username and password if they are configured.